src/openms_gui/include/OpenMS/VISUAL/TOPPViewIdentificationViewBehavior.h

```diff
@@ -36,6 +36,7 @@
 
 #include <OpenMS/METADATA/SpectrumSettings.h>
 #include <OpenMS/VISUAL/LayerData.h>
+#include <OpenMS/VISUAL/BehaviorBase.h>
```

Comment:
Filename and classname differ. please unify.

(and add the new files to the PR) :)

Reply:
Should we really put TOPPView in front of all those? TV would be much shorter. maybe even a namespace could work.
what is your preference?

Reply:
TV sounds good.
Its TOPPView specific, so `BehaviourBase` is a little too general.
But `TVBehaviourBase` should do.